src/patterns/molecules/details/details-custom.hbs

```diff
@@ -0,0 +1,45 @@
+---
+name: Custom expand and collapse
+description: The functions driving the details and summary component can be called independantly
+order: 3
+notes: |
+    - Styling must be provided seperately
+    - Details can be initialized using `Mzp.Details.init(selector, options)`, where `options` is an Object with the following properties.
+      - `onDetailsOpen` [Function] callback when a details is opened (optional).
+      - `onDetailsOpen` [Function] callback when a details is closed (optional).
+---
+
+<style>
+.demo-section .is-closed[aria-hidden=true] { display: none; }
```

Comment:
I'm not sure what you mean here.

Reply:
I was questioning why this needs to be an inline style, which is something we don't generally encourage. I guess I don't yet fully understand why the custom option doesn't have any basic CSS once it's interactive to cover hiding/showing. Maybe just add a note that the inline styles are for demo purposes only, and should be included in the website's regular stylesheet?

Reply:
I understand now thanks 😃 

I avoided defining `.is-closed[aria-hidden=true] { display: none; }` to the global styles because of the possibility of it interfering with a transition. But it's not obvious the demo is working without it. 

The demo seems to not be communicating what I want it to, I'll think about it for a day and see if I can improve it 📈